### ISSUE TITLE
BadImageFormatExpection when Package size increased Xamarin Forms UWP

### ISSUE THREAD

**Issue by srrin** [JamesNK/Newtonsoft.Json#2728]
Hi all,
I am developing a Xamarin Forms application, which refers multiple (.Netstandard) DLLs in my project as library and nuget references.
When the package size increases or referred DLLs size increase, we are facing weird issues when trying to serialize or deserialize json.

We are getting 356: System. Badimageformatexception (Format of the executable (.exe) or library (.dll) is invalid) when trying to serialize or deserialize. I have checked the platform and tried setting everything to x86 or x64 but still it is not working. When we remove few referenced DLLs and when the package size decreased, serialize or deserialize works as expected. 

This issue is specific to Xamarin Forms UWP and not reproduced in IOS / Android

Is there any suggestion we can take it forward

Thanks

**Comment by elgonzo** [comment]
This seems to be related to the Xamarin runtime and not so much with Newtonsoft.Json itself. I find it difficult to believe that Newtonsoft.Json could possibly be responsible for a BadImageFormatException that is related to package size and/or the number of DLLs/dependencies. Hence, have you already tried checking the [Xamarin.Forms issue tracker](https://github.com/xamarin/Xamarin.Forms/issues) for existing reports regarding your issue and/or tried filing an issue report there?

**Comment by srrin** [comment]
Yes I checked, I didnt find any thread related to this. Serialization/ Deserialization failing only when the packages size increases, if we exclude any DLL reference or decrease the package size, then everything is working fine. Seems some strange issue.
